events/bus.go

```diff
@@ -100,6 +111,7 @@ func (bus *EventBus) Publish(event Event) {
 	bus.lock.Lock()
 	defer bus.lock.Unlock()
 	log.Debugf("event: %v", event)
+	collector.WithLabelValues(event.Code.String(), event.Source).Inc()
```

Comment:
So this is the only collector required for tracking events flowing through the bus? I can dig it.